src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -91,88 +153,42 @@ const Index = ({
+                />
 
                 {
-                    selectedOption.value && availableSearchOptions[selectedOption.value].searchGroups
-                        .filter(searchGroup => searchGroup.unique)
-                        .map(({ searchForm, formId }) => {
-                            const name = searchForm.getElements()[0].formName;
-                            return (
-                                <Section
-                                    className={classes.searchDomainSelectorSection}
-                                    header={
-                                        <SectionHeaderSimple
-                                            containerStyle={{ paddingLeft: 8, borderBottom: '1px solid #ECEFF1' }}
-                                            title={i18n.t('Search {{name}}', { name })}
-                                        />
-                                    }
+                    searchStatus === searchPageStatus.NO_RESULTS &&
+                    <Modal position="middle">
+                        <ModalTitle>Empty results</ModalTitle>
+                        <ModalContent>There was no item found</ModalContent>
+                        <ModalActions>
+                            <ButtonStrip end>
+                                <Button
+                                    onClick={closeModal}
+                                    primary
+                                    type="button"
                                 >
-                                    <div className={classes.searchRow}>
-                                        <div className={classes.searchRowSelectElement}>
-                                            {
-                                                forms[formId] &&
-                                                <Form
-                                                    formFoundation={searchForm}
-                                                    id={formId}
-                                                />
-                                            }
-                                        </div>
-                                    </div>
-                                    <div className={classes.searchButtonContainer}>
-                                        <Button onClick={() => {}}>
-                                            Find by {name}
-                                        </Button>
-                                    </div>
-                                </Section>
-                            );
-                        })
+                                    Search Again
+                                </Button>
+                            </ButtonStrip>
+                        </ModalActions>
+                    </Modal>
                 }
+
+                {
+                    searchStatus === searchPageStatus.ERROR &&
+                    <h1 style={{ color: 'red' }}>{searchResultsErrorMessage}</h1>
```

Comment:
The idea of this state is that when there is a network error or unknown error of some sort we can let the user know. In the next PR I am styling this further.